src/etc/inc/certs.inc

```diff
@@ -513,6 +549,12 @@ function cert_get_subject_hash($crt) {
 	return $inf_crt['subject'];
 }
 
+function cert_get_hash($crt) {
+	$str_crt = base64_decode($crt);
```

Comment:
No need to make this variable for a single use, combine the two lines `openssl_x509_parse(base64_decode($crt));`